src/libs/installer/httpthreadworker.cpp

```diff
@@ -37,10 +37,7 @@ void HttpThreadWorker::process(const QByteArray& data, const QString& url, const
     } );
     void (QNetworkReply::*errorSignal)(QNetworkReply::NetworkError) = &QNetworkReply::error;
     connect(reply, errorSignal, [this, reply]() {
-        QString status = QString::fromLatin1("ERROR");
         qWarning() << "framework | HttpThreadWorker::process | Failed to send POST";
-
-        m_finished++; reply->deleteLater();
```

Comment:
Did the deleteLater need to go too for this to work?
Is the reply being deleted elsewhere as well?

Reply:
Good question, I'll try re-adding the `reply->deleteLater()` and see if it still works

Reply:
Keeping it in there doesn't seem to break anything, so I put it back in there. Also split it into two lines in the finish callback so that it's clearer that these are two operations.